### PR TITLE
Remove completion from uninstall item package

### PR DESCRIPTION
**Pull request by hiltonc**

### CODE REVIEW

File: LDSContentTests/XCTAsserts.swift [hunk 1]
```diff
@@ -22,6 +22,13 @@
 
 import XCTest
 
+func XCTAssertThrows<T>(@autoclosure expression: () throws -> T, _ message: String = "", file: StaticString = #file, line: UInt = #line) {
```
Comment (nickmshelley) [inline LDSContentTests/XCTAsserts.swift]: Were you meaning to use this somewhere?

**Comment by nickmshelley** [comment]
Nice, I didn't think to just get rid of the result and make it a throwing function. Do you like it better or do you prefer the consistency of the completion?

File: LDSContentTests/XCTAsserts.swift (hunk 1 above)
Comment (hiltonc) [inline LDSContentTests/XCTAsserts.swift]: Oops, left over from something else. I removed it.

**Comment by hiltonc** [comment]
I don't think the consistency buys us anything. I can see a future where uninstall for some reason does need to happen async, but we can change the API if that ever happens.

**Pull request merged**